src/components/VerificationOverlay.js

```diff
@@ -0,0 +1,34 @@
+const VerificationOverlay = () => {
+  return (
+    <div className="absolute p-2">
+      <div className="space-y-6 w-full bg-neutral-100 p-9 rounded-2xl ">
+        <div className="text-blue-600 dark:text-white text-3xl font-extrabold">
+          Account Verification
+        </div>
+        <p>
+          A verification link has been sent to{" "}
+          <span className="text-blue-600">augustine@example.com.</span> Please
```

Comment:
Please change this to be the prop `{email}`

Reply:
Also remove `please check your email` to another line

Reply:
Oh Ok I was actually follow accordingly to the design